internals/cli/cmd_help.go

```diff
@@ -219,12 +219,16 @@ the system that is running them.
 `)
 	pebbleUsage               = "Usage: {{.ProgramName}} <command> [<options>...]"
 	pebbleHelpCategoriesIntro = "Commands can be classified as follows:"
-	pebbleHelpAllFooter       = "Set the PEBBLE environment variable to override the configuration directory \n" +
-		"(which defaults to " + defaultPebbleDir + "). Set PEBBLE_SOCKET to override \n" +
-		"the unix socket used for the API (defaults to $PEBBLE/.pebble.socket).\n" +
-		"\n" +
-		"For more information about a command, run '{{.ProgramName}} help <command>'."
-	pebbleHelpFooter = "For a short summary of all commands, run '{{.ProgramName}} help --all'."
+
+	// can be overridden in derivative projects
+	PebbleHelpAllFooterText = strings.TrimSpace(`
```

Comment:
Just a couple of comments: 

* Can we come up with a better name for this? Maybe remove the "Pebble", maybe `FooterHelp`? Or maybe get more concrete, like `EnvVarsHelp`?
* What will other projects actually change this to? Would it be better to make the names of the environment variables customisable, rather than the entire paragraph?

Reply:
Maybe `EnvVarsHelp` is too specific, but something in between `FooterHelp` and `EnvVarsHelp`?

Reply:
After reading through it again, I decided to go with `HelpHeader` (if we allow overriding the footer, we might also want to allow overriding the header) and `HelpFooter`.

This then makes it read from another package as `cli.HelpHeader` and `cli.HelpFooter` (it's the header of the help and the footer of the help) vs `cli.HeaderHelp` and `cli.FooterHelp` (it's not the help of the header, and not the help of the footer).